### PR TITLE
Test Drupal 8 with big_pipe enabled

### PR DESCRIPTION
**Pull request by jhedstrom**
- Addresses #258

### CODE REVIEW

File: features/messages.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@api @d8 @arb
```
Comment (pfrenssen) [inline features/messages.feature]: @arb?

Comment (jhedstrom) [inline features/messages.feature]: Lol, I was using that for local testing as the behat command seems broken when trying to run line numbers of features combined with `--profile` :)

Comment (arb) [inline features/messages.feature]: Yes? :smile:

Comment (jhedstrom) [inline features/messages.feature]: Ha!

Comment (pfrenssen) [inline features/messages.feature]: Hahaha! Hello there!

Comment (arb) [inline features/messages.feature]: This LGTM. I say :shipit:

**Pull request merged**